src/lib/functions/netlify-function.ts

```diff
@@ -156,8 +153,8 @@ export default class NetlifyFunction {
       return null
     }
 
-    // @ts-expect-error TS(2339) FIXME: Property 'schedule' does not exist on type 'Netlif... Remove this comment to see the full error message
-    return getNextRun(this.schedule)
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    return getNextRun(this.schedule!)
```

Comment:
that'd be pretty much a duplicate of `this.isScheduled()` - I prefer disabling the lint rule over that suggestion

Reply:
I just think we should try to avoid disabling rules on an ad-hoc basis as much as possible. We either have a rule that we think is useful and adhere to it everywhere, or we disable it altogether. And I personally think this is one we should keep, because the non-null operator can be dangerous.

What if you change `getNextRun()` to this and get rid of `isScheduled()` altogether?

```ts
  async getNextRun() {
    await this.buildQueue

    if (!this.schedule) {
      return
    }

    return getNextRun(this.schedule)
  }
```

Reply:
Ah, wait. We use `isScheduled()` elsewhere.